apps/bot/db/actions/users.ts

```diff
@@ -143,3 +143,7 @@ export const removePointsFromUser = async (
   type: keyof typeof POINTS_REWARDS,
   trx: TransactionDB | KyselyDB = db,
 ) => updatePointsBySum(userId, -POINTS_REWARDS[type], trx)
+
+export async function getCorrectAnswersCount(userId: string) {
+  return db.selectFrom('users').select(['answersCount']).executeTakeFirst()
```

Comment:
I may be dumb here, but this query doesn't look like it used `userId`.

Reply:
I'll fix it

Reply:
I tested & fixed it now